olm/2.12.2/openebsoperator.v2.12.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,634 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: openebsoperator.v2.12.2
+  namespace: default
+  annotations:
+    capabilities: Basic Install
+    categories: "Storage"
+    containerImage: index.docker.io/openebs/helm-operator:v0.0.8
+    createdAt: 2021-09-24T16:48:09Z
+    repository: https://github.com/openebs/helm-operator
+    certified: "false"
+    support: https://slack.openebs.io/
+    alm-examples: |
+      [
+        {
+          "apiVersion": "openebs.io/v1alpha1",
+          "kind": "OpenEBSInstallTemplate",
+          "metadata": {
+            "name": "oebs",
+            "namespace": "openebs"
+          },
+          "spec": {
+            "rbac": {
+              "create": true,
+              "pspEnabled": false,
+              "kyvernoEnabled": false
+            },
+            "serviceAccount": {
+              "create": true,
+              "name": "openebs-maya-operator"
+            },
+            "imagePullSecrets": [],
+            "release": {
+              "version": "2.12.2"
+            },
+            "legacy": {
+              "enabled": true
+            },
+            "image": {
+              "pullPolicy": "IfNotPresent",
+              "repository": ""
+            },
+            "apiserver": {
+              "enabled": true,
+              "image": "quay.io/openebs/m-apiserver",
+              "imageTag": "2.12.2",
+              "replicas": 1,
+              "ports": {
+                "externalPort": 5656,
+                "internalPort": 5656
+              },
+              "sparse": {
+                "enabled": "false"
```

Comment:
Should this be without quotes

Reply:
getting this value from https://github.com/openebs/charts/blob/15ad1ccf319e935cf68243c2459628186df48494/charts/openebs/values.yaml#L42